GoogleForCreators/web-stories-wp
# Editor: Border Radius panel redesign

## Context

Updating the design panel to match the new designs: [Designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3471%3A4831).

## Summary

Updates the `Corner Radius` or `Border Radius` panel.

## Relevant Technical Choices

- reduced the min width of the `<Input />` since we needed smaller inputs from the design system

## To-do

no

## User-facing changes



|Before|After|
|--|--|
|![corner-radius-panel-before](https://user-images.githubusercontent.com/22185279/109889560-da794b00-7c42-11eb-81a7-c784338758e7.gif)|![border-radius-panel](https://user-images.githubusercontent.com/22185279/109889470-aaca4300-7c42-11eb-9e3e-6c35be2339f7.gif)|


## Testing Instructions



1. Go to editor
2. Click on an element
3. Go to `Corner Radius` panel 
4. Mess with inputs, change the radiuses, try keyboard navigation too
5. Blurring the inputs should still change the border radiuses
6. All functionality should work the same as before

### QA






This PR can be tested by following these steps:

1. Go to editor
2. Click on an element
3. Go to `Corner Radius` panel 
4. Mess with inputs, change the radiuses, try keyboard navigation too
5. Blurring the inputs should still change the border radiuses
6. All functionality should work the same as before


### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6476
